tests/stubs/stub.phpstub

```diff
@@ -1,1079 +0,0 @@
-<?php
```

Comment:
Why is this file deleted? This is why CI is red I suppose.

Reply:
Argh, I am truly sorry. Something must have gotten wrong.
I fixed it by restoring the file and adding the license header via https://github.com/nextcloud/ldap_contacts_backend/pull/240/commits/0a1e337ca708dc3de6f7695a719aee90ed9f4c4c